src/src.pro

```diff
@@ -28,6 +30,8 @@ PUBLIC_HEADERS += \
     $$PWD/emailmessage.h \
     $$PWD/emailaccountsettingsmodel.h \
     $$PWD/emailaccount.h \
+    $$PWD/emailcontactmodel.h \
+    $$PWD/emailcontactproxymodel.h \
```

Comment:
Do you really need these public, that is provided to external c++ projects?

Reply:
done